bot/src/plugins/hangman.cr

```diff
@@ -134,10 +139,10 @@ class Hangman
     end
   end
 
-  def start_game msg, list
+  def start_game msg, list, guess_max
     unless GAMES.has_key? msg.channel.name
       if Game::WORDLISTS.has_key? list
-        GAMES[msg.channel.name] = Game.new list
+        GAMES[msg.channel.name] = Game.new list, [guess_max, Game::DEFAULT_GUESS_MAX].min
```

Comment:
Don't need to allocate an array here, just use a tuple: `{guess_max, Game::DEFAULT_GUESS_MAX}.min`